console/Makefile

```diff
@@ -7,3 +7,10 @@ package:
 	tar -czf build/console.tgz bin lib www package.json
 
 test: npm_test
+
+coverage: npm_install
```

Comment:
Maybe move the 'implementation' of this rule into Makefile.common, and rename it node_coverage ?

Reply:
The issue there is that not all the subdirs that use Makefile.common have a package.json yet (and so can't use npm_install).